tests/test_comment_spell_check.py

```diff
@@ -26,13 +26,12 @@ def test_basic(self):
             ],
             stdout=subprocess.PIPE,
         )
-        if runresult.returncode:
-            self.fail("\nBasic Test: FAIL")
-            output_string = str(runresult.stdout)
-            print("\nTest output:", output_string)
+        self.assertEqual(
+            runresult.returncode, 0, "Basic test FAIL: " + str(runresult.stdout)
```

Comment:
What is displayed by unit test when an assert fails? Is the name of the current test and the "FAIL" status really needed?

Reply:
New commit gets rid of that and reduces the output message when a test fails.